man/systemd.network.xml

```diff
@@ -776,6 +776,31 @@ Table=1234</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>IPv4ReversePathFilter=</varname></term>
+        <listitem>
+          <para>Configures IPv4 Reverse Path Filtering. Reverse Path Filtering is used to prevent packets that arrived
+          via one interface from leaving via a different interface. When a machine with reverse path filtering enabled
+          receives a packet, the machine will first check whether the source of the received packet is reachable through
+          the interface it came in. If it is routable through the interface which it came, then the machine will accept the packet.
+          If it is not routable through the interface which it came, then the machine will drop that packet. Takes one of
+          <literal>no</literal> No source validation.
+          <literal>strict</literal> In <literal>strict</literal> mode each incoming packet is tested against the FIB and
+          if the incoming interface is not the best reverse path, the packet check will fail. By default failed packets are discarded.
+          <literal>loose</literal> In <literal>loose</literal> mode each incoming packet's source address is tested against the FIB.
```

Comment:
Something is wrong with capitalization here. I'd expect something like `<literal>no</literal> — no source validation`.